docs/src/examples/components/Label/Variations/LabelExampleColor.shorthand.tsx

```diff
@@ -1,10 +1,11 @@
 import * as React from 'react'
 import { Label, Provider } from '@stardust-ui/react'
+import * as _ from 'lodash'
 
 const LabelExampleColor = () => (
   <Provider.Consumer
-    render={({ siteVariables: { colorScheme } }) =>
-      Object.keys(colorScheme).map(color => (
+    render={({ siteVariables: { primitiveColors, emphasisColors, naturalColors } }) =>
+      _.keys({ ...primitiveColors, ...emphasisColors, ...naturalColors }).map(color => (
```

Comment:
Extracting only the colors we had previously in the example

Reply:
`DividerExample` loops through emphasis and natural colors only (no primitive) is the difference intentional?

Reply:
`TextExample` as well